samples/template/src/module/module.yml

```diff
@@ -0,0 +1,136 @@
+# module name, required
+name: ${oc.env:MODULE_NAME, 'template'}
+
+# base module parameters
+parameters:
+  # DevMode (hot Python code reload on file change)
+  dev_mode: True
+  # Allows sending shutdown message to the module with the auth-key (see below)
+  shutdown_auth: "shutdown"
+  # enable OpenTelemetry
+  telemetry:
+    sampling_period: 100
+    root_span_name: pipeline
+    provider: jaeger
+    provider_params:
+      service_name: demo-pipeline
+      endpoint: jaeger:6831
+
+  # pipeline processing frame parameters
+  frame:
+    width: 1280
+    height: 720
+    # Add paddings to the frame before processing
+  output_frame:
+    codec: raw-rgba
+  # custom drawing on frames function
+  draw_func:
+    module: src.module.overlay_draw_spec
+    class_name: Overlay
+    #kwargs:
+    rendered_objects:
+      peoplenet:
+        person:
+        face:
+
+
+# pipeline definition
+pipeline:
+  # source definition
```

Comment:
I'm not sure we should provide source/sink options here (neither commented nor default).

Reply:
Removed the source / sink definitions.